wurst/objects/items/MedallionOfCourage.wurst

```diff
@@ -80,17 +24,13 @@ function onCast(unit caster, item target)
         simError(owner, "You need to be a basic troll (mage, priest, thief, scout, gatherer) in order to use this item")
         return
 
-    _superSubUpgradeId = superSubUpgrade.get(unitId)
-
-    if _superSubUpgradeId != 0
-        if GetUnitAbilityLevel(caster, _superSubUpgradeId) == 0 // Checking if the ability already exist so we don't waste a medallion
-            string medalHint = "{0} {1}".format(caster.getProperName().color(GOLD_COLOR), "acquired a super class choice! Check your hero skill and reach level 9 to use it.".color(GENERAL_COLOR))
-            printTimedToPlayer(medalHint, 5, owner)
-            SetPlayerAbilityAvailable(owner, _superSubUpgradeId, false)
-            caster.addAbility(_superSubUpgradeId)
-            target.remove()
-        else
-            simError(owner, "This unit already used the medallion of Courage")
+    if GetPlayerTechResearched(caster.getOwner(), UPGD_MEDALLION_COURAGE, true) == false
+        SetPlayerTechResearched(caster.getOwner(), UPGD_MEDALLION_COURAGE, 1)
+        let medalHint = ("{0} acquired a super class choice! Check your hero skill and reach level 9 to use it.".color(GENERAL_COLOR)).format(caster.getProperName().color(GOLD_COLOR))
```

Comment:
I this might mess up the color though, because `{0}` will close the color tag for both the gold and general color.